image_provisioner/playbooks/roles/aos-ansible/templates/hosts-atomic.j2

```diff
@@ -0,0 +1,22 @@
+[OSEv3:children]
```

Comment:
Please use conditionals where necessary in templates rather than having multiple steps and multiple files.